emb/project/robo24/main.cc

```diff
@@ -0,0 +1,25 @@
+#include <functional>
+#include <inttypes.h>
+#include <map>
+
+#include "emb/network/node/node.hpp"
+#include "emb/network/serialize/bh_cobs.hpp"
+#include "emb/network/transport/serial.hpp"
+#include "emb/project/base/base_bh.hpp"
+#include "emb/project/robo24/robo24_bh.hpp"
+
+int main() {
+    // Create named variables for the objects to prevent
+    // the objects from being destroyed before the node
+    emb::network::serialize::BhCobs serializer;
+    emb::network::transport::Serial transporter;
+    emb::project::base::Base base;
```

Comment:
I'm surprised this actually worked and I got it right, but adding all of the callbacks for transactions from `base.bh` is as easy as adding this and linking against `emb/project/base/base.cc` for the handler definitions. If a different implementation is wanted, just make a new `base.cc` and link against that!